tracer/test/snapshots/Iast.HeaderInjection.AspNetCore2.Vuln.SensitiveValue.verified.txt

```diff
@@ -0,0 +1,89 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/headerinjection,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Tags: {
+      aspnet_core.route: iast/headerinjection,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/headerinjection,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/HeaderInjection?useValueFromOriginHeader=False,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "HEADER_INJECTION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "myName",
+            "source": 0
+          },
+          {
+            "value": ": "
+          },
+          {
+            "value": ":bearer secret",
```

Comment:
I assumed this should be redacted if it's sensitive, no? 🤔

Reply:
In this test, redaction is disabled. The enabled version is in Iast.HeaderInjection.AspNetCore2.Vuln.SensitiveValue.RedactionEnabled.verified.txt